config/initializers/deprecations.rb

```diff
@@ -1 +1,28 @@
 ActiveSupport::Deprecation.behavior = :silence if Rails.env.production?
+
+Foreman::Application.configure do |app|
+  config.after_initialize do
+    next if Foreman.in_rake?
+    next unless Rails.env.production?
+    next unless ComputeResource.all.unscoped.where(type: 'Foreman::Model::Rackspace').any?
+    blueprint = NotificationBlueprint.find_by_name('feature_deprecation')
+    next unless blueprint
+    message = UINotifications::StringParser.new(blueprint.message, {feature: 'Rackspace Compute Resource', version: '2.0'}).to_s
```

Comment:
the version should be 2.1 - deprecations state the version in which something will be removed.